golang/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Golang versions
 
-All golang services (with some exceptions) at Clever should be on go-1.5.
+All golang services (with some exceptions, like Heka) at Clever should be on go-1.5.
```

Comment:
As far as @cgclever and I know the only exception is Heka. I'm going to note that all "services written by Clever" should be Go1.5 and acknowledge that 3rd party exceptions exist